test/functionalTest/cases/1587_q_query_has_type/q_query_has_type.test

```diff
@@ -1,186 +0,0 @@
-# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
The `+type` doesn't make sense as in NGSIv2 all entities will have type (there is a restriction on IDs for a minimum length of 1 chraacter). Thus, this .test is removed.

NTC (just informative).